discovery/src/main/scala/akka/discovery/SimpleServiceDiscovery.scala

```diff
@@ -41,13 +41,30 @@ object SimpleServiceDiscovery {
     }
   }
 
-  /** Resolved target host, with optional port and protocol spoken */
-  final case class ResolvedTarget(host: String, port: Option[Int]) {
+  /**
+   * Resolved target host, with optional port and the IP address.
+   * The address may be used for cluster bootstrap.
```

Comment:
Should we explain that the address is expected to be an IP while the host may be a hostname or ip?

Reply:
"expected"? 😕 It would be better if this were "enforced", i.e with a specific type.

Reply:
Good point, could you follow up on this @eed3si9n and do the comment rewording as well as I requested there?